multi-agent/tests/unit/resources/conftest.py

```
@@ -0,0 +1,428 @@
+                continue
+            if resource_type and doc.type != resource_type:
+                continue
+            result.append(doc)
+        return result
+
+    def find_visible_for_identity(
+        self,
+        *,
+        identity: Optional[Identity],
+        category: Optional[str],
+        resource_type: Optional[str],
```

Ruff (0.16.1)

[warning] Unused method argument: `resource_type`

(ARG002)